sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Window.java

```diff
@@ -603,7 +603,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
           .include(windowFn)
           .addIfNotNull("allowedLateness", allowedLateness);
 
-      if (trigger != null) {
+      if (trigger != null && !(trigger instanceof DefaultTrigger)) {
```

Comment:
`null` is not an instance of anything so this can be:
`if (!(trigger instanceof DefaultTrigger))`